core/players-handbook-2024/races/race-goliath.xml

```diff
@@ -0,0 +1,149 @@
+			<grant type="Size" id="ID_SIZE_MEDIUM" />
+			<stat name="innate speed" value="35" bonus="base" />
+
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language (Goliath)" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Language (Goliath)" supports="Standard||Exotic" number="2" />
+
+			<grant type="Racial Trait" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY" requirements="!ID_WOTC_PHB24_RACIAL_TRAIT_REPLACEMENT_GOLIATH_GIANT_ANCESTRY" />
+			<grant type="Racial Trait" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_LARGE_FORM" level="5" requirements="!ID_WOTC_PHB24_RACIAL_TRAIT_REPLACEMENT_GOLIATH_LARGE_FORM" />
+			<grant type="Racial Trait" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_POWERFUL_BUILD" requirements="!ID_WOTC_PHB24_RACIAL_TRAIT_REPLACEMENT_GOLIATH_POWERFUL_BUILD" />
+		</rules>
+	</element>
+
+	<element name="Giant Ancestry" type="Racial Trait" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_RACIAL_TRAIT_GOLIATH_GIANT_ANCESTRY">
+		<description>
+			<p>You are descended from Giants. Choose one of the following benefits—a supernat­ural boon from your ancestry; you can use the chosen benefit a number of times equal to your Proficiency Bonus, and you regain all expended uses when you finish a Long Rest:</p>
+			<ul class="unstyled">
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Cloud’s Jaunt (Cloud Giant).</strong> As a Bonus Action, you magically teleport up to 30 feet to an unoccu­pied space you can see.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Fire’s Burn (Fire Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal ldlO Fire damage to that target.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Frost’s Chill (Frost Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal ld6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Hill’s Tumble (Hill Giant).</strong> When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Stone’s Endurance (Stone Giant).</strong> When you take damage, you can take a Reaction to roll ldl2. Add your Constitution modifier to the number rolled and reduce the damage by that total.</li>
+				<li style="margin-left:1em;text-indent:-1em;"><strong>Storm’s Thunder (Storm Giant).</strong> When you take damage from a creature within 60 feet of you, you can take a Reaction to deal ld8 Thunder dam­age to that creature.</li>
```

Comment:
seems like anywhere 1d# was copied from your source, it translated it into letters instead of numbers

```suggestion
				<li style="margin-left:1em;text-indent:-1em;"><strong>Cloud’s Jaunt (Cloud Giant).</strong> As a Bonus Action, you magically teleport up to 30 feet to an unoccu­pied space you can see.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Fire’s Burn (Fire Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal 1d10 Fire damage to that target.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Frost’s Chill (Frost Giant).</strong> When you hit a target with an attack roll and deal damage to it, you can also deal 1d6 Cold damage to that target and reduce its Speed by 10 feet until the start of your next turn.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Hill’s Tumble (Hill Giant).</strong> When you hit a Large or smaller creature with an attack roll and deal damage to it, you can give that target the Prone condition.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Stone’s Endurance (Stone Giant).</strong> When you take damage, you can take a Reaction to roll 1d12. Add your Constitution modifier to the number rolled and reduce the damage by that total.</li>
				<li style="margin-left:1em;text-indent:-1em;"><strong>Storm’s Thunder (Storm Giant).</strong> When you take damage from a creature within 60 feet of you, you can take a Reaction to deal 1d8 Thunder dam­age to that creature.</li>
```